tests/arch/arm64/arm64_gicv3_its/src/main.c

```diff
@@ -77,8 +77,7 @@ static void test_gicv3_its_connect(void)
 		remain = event_id - ITS_TEST_NUM_ITES;
 	}
 }
-
```

Comment:
nit: keep the blank line here, it'll make the diff cleaner

Reply:
Fixed